src/conv/k-g/region_list.cpp

```diff
@@ -49,16 +49,53 @@ Bot& RegionList::addRegion
 }
 
 
+void RegionList::setAttributes
+(
+    const std::string&                        name,
+    const std::map<std::string, std::string>& attributes
+) {
+    m_list[name].attributes = attributes;
+}
+
+static void  writeAttributes
+(
+    rt_wdb* wdbp,
+    const char* name,
+    const std::map<std::string, std::string>& attributes
+) {
+
+    struct rt_db_internal     region_internal;

```

Comment:
Hmm, usually a `RT_DB_INTERNAL_INIT(&region_internal)` and `rt_db_free_internal(&region_internal)` is needed.

Reply:
I checked some other codes the RT_DB_INTERNAL_INIT wasn't used, but I did find that rt_db_free_internal(&region_internal) is used all the time. I will modify it

Reply:
I think because the rt_db_internal is intialized using RT_DB_INTERNAL_INIT  inside the function rt_db_get_internal()